sdk/identity/identity/src/credentials/browserCustomizatioOptions.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like this file name has a typo for "Customization" (missing the n)

Reply:
ooh

Reply:
done